packages/table/src/Table.tsx

```diff
@@ -20,6 +20,7 @@ export const Table = <TData extends TableData>(props: PropsWithChildren<TablePro
   return (
```

Comment:
You want me to add some tsdoc for the Table component here?

Reply:
sure if you have the time :)